android-svc-lib.sh

```diff
@@ -11,6 +11,13 @@ export g_green=$(printf '%s\n' '\033[0;32m' | sed -e 's/[\/&]/\\&/g')
 export g_yellow=$(printf '%s\n' '\033[0;33m' | sed -e 's/[\/&]/\\&/g')
 export g_nc=$(printf '%s\n' '\033[0m' | sed -e 's/[\/&]/\\&/g') # no color
 
+###
+# init g_repoUrl value
+# GLOBALS:
+#  l_targetDir
```

Comment:
`l_targetDir` must be a mistake I'll have to look into. I'm not sure how I intended this to work, but all variables starting with `l_` are supposed to be local variables and all variables starting with `g_` are supposed to be global variables.